_proto/hapi/chart/metadata.proto

```diff
@@ -80,4 +80,7 @@ message Metadata {
 	// TillerVersion is a SemVer constraints on what version of Tiller is required.
 	// See SemVer ranges here: https://github.com/Masterminds/semver#basic-comparisons
 	string tillerVersion = 15;
+
+	// Annotations is an unstructured key value map for user to store arbitrary data.
```

Comment:
- [ ] Instead:
  > Annotations are additional mappings uninterpreted by Tiller, made available for inspection by other applications.